### ISSUE TITLE
Enable saving and loading

### ISSUE THREAD

**Issue by MarkSill** [MarkSill/social#12]
I've written a save/load system into the game (8c0ea98bfb1c0fd38de9e0c6b53c68834d6b57d5), but it is not currently accessible. I should probably make it so people can use it.

**Comment by MarkSill** [comment]
I will likely be making this possible to use on Friday (4/29/16). Hopefully I'll be able to get it working straight away.

**Comment by MarkSill** [comment]
Saving is working as of 114d843cd8aeea66c0e8e95dcc8106e519856ab5. I'm having a bit of trouble with loading the files properly (it loads them but also keeps instances from the last time), but it should be done before too long. If not today, then Tuesay (5/3/16).

**Comment by MarkSill** [comment]
An example of what save files look like:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<root>
  <instance type="InstanceGame">
    <name>Game</name>
    <instance type="InstanceWorld">
      <name>World</name>
      <gravX>0.0</gravX>
      <gravY>-9.8</gravY>
      <instance type="InstanceScript">
        <name>Script</name>
        <enabled>true</enabled>
        <code>while true do
	wait(100)
	local block = Instance:create('rectangle', game:findChild('World'))
	block.position = Vector2.new(15, 20)
end</code>
        <instance type="InstanceClientScript">
          <name>ClientScript</name>
          <enabled>true</enabled>
          <code>print("test")</code>
        </instance>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>-0.00328331825034812</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>0.026589968519918333</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>8.013595509800727E-4</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>0.06171896422016969</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>0.00444478402254568</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>0.00478112293142136</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>0.0038127103593593166</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>0.004740475018837811</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>0.05726228348098052</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>-0.007723194875187091</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>-0.025270629439127886</rotation>
      </instance>
      <instance type="InstanceRectangle">
        <name>rectangle</name>
        <anchored>false</anchored>
        <visible>true</visible>
        <mass>1.0</mass>
        <density>1.0</density>
        <elasticity>0.0</elasticity>
        <friction>0.2</friction>
        <colorR>1.0</colorR>
        <colorG>1.0</colorG>
        <colorB>1.0</colorB>
        <colorA>1.0</colorA>
        <rotationLocked>false</rotationLocked>
        <rotation>-0.12112364845851291</rotation>
      </instance>
    </instance>
    <instance type="InstancePlayers">
      <name>Players</name>
      <maxPlayers>4</maxPlayers>
    </instance>
  </instance>
</root>

```

**Comment by MarkSill** [comment]
Many bugs fixed, along with loading games, in a2e060db65fddec796ef8f3cf65d69f633dc5b8e.
